packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -210,6 +211,18 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// If null then [kMaxFlingVelocity] is used.
   double? maxFlingVelocity;
 
+  /// Whether the drag callbacks should only be dispatched when the drag threshold
+  /// has been met.
+  ///
+  /// If true, the drag callbacks will only be dispatched when this recognizer has
+  /// won the arena and the drag threshold has been met.
+  ///
+  /// If false, the drag callbacks will be dispatched when this recognizer has won
+  /// the arena.
```

Comment:
Nit: Maybe say "...will be dispatched immediately when..."

At first glance this seems to say almost the same thing as the paragraph above.